cleanlab/count.py

```diff
@@ -43,6 +42,8 @@
     get_num_classes,
     is_torch_dataset,
     is_tensorflow_dataset,
+    int2onehot,
+    _binarize_pred_probs_slice,
```

Comment:
Need to update documentation to describe what shape `_compute_confident_joint_multi_label()` returns, especially with `return_indices_of_off_diagonals`